pynxtools/dataconverter/helpers.py

```diff
@@ -264,14 +264,12 @@ def convert_data_dict_path_to_hdf5_path(path) -> str:
     return hdf5path
 
 
-def is_value_valid_element_of_enum(value, elem) -> Tuple[bool, list]:
+def is_value_valid_element_of_enum(value, elist) -> Tuple[bool, list]:
     """Checks whether a value has to be specific from the NXDL enumeration and returns options."""
-    if elem is not None:
-        has_enums, enums = nexus.get_enums(elem)
-        if has_enums and (
-            isinstance(value, list) or value not in enums[0:-1] or value == ""
-        ):
-            return False, enums
+    for elem in elist:
```

Comment:
We may want to check if elist is None or an empty list

Reply:
I checked this already and elist is always a list. If there are not elements is just the empty list.

We also use it like this here:
https://github.com/FAIRmat-NFDI/pynxtools/blob/48314117452e21fd3e8003f220a0d9556fd350ab/pynxtools/dataconverter/helpers.py#L110